pkgs/development/tools/icr/default.nix

```
@@ -1,15 +1,26 @@
-{ stdenv, lib, fetchFromGitHub, crystal, shards, makeWrapper, pkg-config, which
-, openssl, readline, libyaml, zlib }:
+{ stdenv
+, lib
+, fetchFromGitHub
+, crystal
+, shards
+, makeWrapper
+, pkg-config
+, which
+, openssl
+, readline
+, libyaml
+, zlib
+}:
 
 crystal.buildCrystalPackage rec {
   pname = "icr";
-  version = "0.8.0";
+  version = "0.8.0.20201006";
 
   src = fetchFromGitHub {
     owner = "crystal-community";
     repo = pname;
     rev = "v${version}";
```

```suggestion
    rev = "8c57cd7c1fdf8088cb05c1587bd6c40d244a8a80";
```
This is what you had in mind, there's no `v0.8.0.20201006` tag, see: https://github.com/crystal-community/icr/tags

---

Messed up the merge, thanks.